cmd/oci/oci.go

```diff
@@ -237,6 +252,38 @@ func ProcessPlatformAndConfig(plat string, envName string) (hcl2nix.OCIArtifact,
 	return artifact, plat, nil
 }
 
+func modifyDockerfileWithTag(path, tag string, devDeps bool) error {
+	var dockerfilePath string
+	if path != "" {
+		dockerfilePath = path + "/Dockerfile"
+	} else {
+		dockerfilePath = "./Dockerfile"
+	}
+
+	_, err := os.Stat(dockerfilePath)
```

Comment:
Do we need to stat? 
Because if the os.Open fails, we can figure out that the file doesn't exist.